sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -535,3 +538,75 @@ def _to_generated(self) -> GeneratedConfigurationSnapshot:
             retention_period=self.retention_period,
             tags=self.tags,
         )
+
+
+class ConfigurationSettingPropertiesPaged(PageIterator):
+    """An iterable of ConfigurationSetting properties."""
+
+    etag: str
+    """The etag of current page."""
+    
+    def __init__(self, command: Callable, **kwargs):
+        super(ConfigurationSettingPropertiesPaged, self).__init__(
+            self._get_next_cb,
+            self._extract_data_cb,
+            continuation_token=kwargs.get("continuation_token"),
+        )
+        self._command = command
+        self._key = kwargs.get("key")
+        self._label = kwargs.get("label")
+        self._accept_datetime = kwargs.get("accept_datetime")
+        self._select = kwargs.get("select")
+        self._cls = kwargs.get("cls")
+        self._deserializer = lambda objs: [ConfigurationSetting._from_generated(x) for x in objs]
+    
+    def _get_next_cb(self, continuation_token):
+        return self._command(
+            key=self._key,
+            label=self._label,
+            accept_datetime=self._accept_datetime,
+            select=self._select,
+            continuation_token=continuation_token,
+            cls=self._cls,
+        )
+
+    def _extract_data_cb(self, get_next_return):
```

Comment:
Can you add the type annotation?

Shouldn't get_next_return a response type?

Reply:
* I think type annotation is not necessary for this private function. 
* `get_next_return ` is a tuple not a response type as I passed [this](https://github.com/Azure/azure-sdk-for-python/pull/34346/files#diff-f1ef905e8030daf11ec8699b887dcad48b91d270810d48b03c1ded0e6c852b9aR231) `cls` to return deserialized data from response and response header.

Reply:
That's the reason I asked for type annotation. :)

Mypy will help us check if we miss any cases.